client/src/component/Header/index.js

```diff
@@ -636,15 +685,17 @@ export default function Header(props) {
                     <FavoriteBorder />
                   </IconButton>
                 </Link>
-                <Link to="/cart">
-                  <IconButton
-                    aria-label="show cart"
-                    color="primary"
-                    className={classes.buttonIcons}
-                  >
-                    <ShoppingCartIcon />
-                  </IconButton>
-                </Link>
+                <Button onClick={handleCartButton}>
+                  <Link to="/cart">
+                    <IconButton
+                      aria-label="show cart"
+                      color="primary"
+                      className={classes.buttonIcons}
+                    >
+                      <ShoppingCartIcon />
+                    </IconButton>
+                  </Link>
+                </Button>
```

Comment:
please remove the button and add on click to the link
```suggestion

                <Link to="/cart" onClick={handleCartButton}>
                  <IconButton
                    aria-label="show cart"
                    color="primary"
                    className={classes.buttonIcons}
                  >
                    <ShoppingCartIcon />
                  </IconButton>
                </Link>

```